app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,32 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @venue = Venue.find(params[:venue_id])
+    @review = Review.new
+    @quantity_collection = [[1,1], [2,2], [3,3], [4,4], [5,5]]
+  end
+
+  def create
+    @quantity_collection = [[1,1], [2,2], [3,3], [4,4], [5,5]]
+    @venue = Venue.find(params[:venue_id])
+    @review = Review.new(review_params)
+    @review.venue = @venue
+    @review.user_id = 1
```

Comment:
You want to set it to the use who is currently signed in, so change this line to `@review.user = current_user`